### DISCUSSION TITLE
Dropdown (BDropdown) doesn't work

### DISCUSSION THREAD

**Discussion by hencework** [cdmoro/bootstrap-vue-3#2233] [Q&A]
Dropdown (BDropdown) doesn't work after page refresh or reload, same issue in your documentation dropdown page. This is our dropdown structure according to your doc : 
&lt;BButtonGroup>
        &lt;BDropdown text="Dropdown">
          &lt;BDropdownItem>Action</BDropdownItem>
          &lt;BDropdownItem>Another action</BDropdownItem>
          &lt;BDropdownItem>Something else here</BDropdownItem>
          &lt;BDropdownDivider />
          &lt;BDropdownItem>Separated link</BDropdownItem>
        &lt;/BDropdown>
&lt;/BButtonGroup>

**Comment by xvaara** [comment]
What browser are you using? Since I tested safari, chrome and firefox and the all seem to work fine and we haven't received any bug reports about this.
